prow/tekton.md

```diff
@@ -0,0 +1,147 @@
+# Using Prow with Tekton Pipeline
+
+The Prow agent type `tekton-pipeline` will allow you to trigger
+execution of
+[Tekton Pipelines](https://github.com/tektoncd/pipeline) 
+with Prow!
+
+1. [Setting up Pipelines with Prow](#setting-up-pipelines-with-prow)
+2. [Configuring a `tekton-pipeline` job](#configuring-a-tekton-pipeline-job)
+
+**Warning**
+
+Prow users may expect to see logs from their `PipelineRuns` be visible in
+Gubernator, however this functionality is not supported yet, and logs must be
```

Comment:
Gubernator is deprecated in favor of Spyglass.  It might be more accurate to note that pipeline prowjobs cannot be decorated with the pod utilities so job logs, metadata, and artifacts are not uploaded to GCS.

Reply:
kk updated!